packages/rich-text/src/RichTextEditor.styles.ts

```diff
@@ -13,6 +13,8 @@ export const styles = {
     borderTop: 0,
     padding: '20px',
     fontSize: tokens.spacingM,
+    fontFamily:
+      '-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol',
```

Comment:
we can get this from F36 tokens - `fontStackPrimary` is the token we'd use

Reply:
We can do this as a separate PR, there are a few other things that can be turned into tokens (and fontSize is currently using a spacing token, oops)

Reply:
Thanks, just added it ;)